lib/Github/Client.php

```diff
@@ -200,7 +201,8 @@ public function api($name)
 
             case 'integration':
             case 'integrations':
-                $api = new Api\Integrations($this);
```

Comment:
You should keep this. Calling integrations should give you the deprecated Integrations class, right?